docs/user/login-alternatives.md

```diff
@@ -0,0 +1,36 @@
+# Kubeapps Login Form
+
+By default, when first visiting Kubeapps, a login form is shown for the user to introduce a personal token for a specific service account:
+
+![Dashboard Login](../img/dashboard-login.png)
+
+You can find more information about access control in Kubeapps in this [document](./access-control.md).
```

Comment:
This line doesn't feel very relevant here and it's already being linked to from other places, so I think we can skip it?